medcat/cdb.py

```diff
@@ -661,6 +663,16 @@ def _ensure_backward_compatibility(config: Config) -> None:
         if 'tagger' in disabled_comps and 'lemmatizer' not in disabled_comps:
             config.general['spacy_disabled_components'].append('lemmatizer')
 
+    @staticmethod
+    def _check_medcat_version(config_data: Dict, log: logging.Logger) -> None:
+        cdb_medcat_version = config_data.get('version', {}).get('medcat_version', 'unknown')
+        if __version__.split(".")[:2] != cdb_medcat_version.split(".")[:2]:
+            log.warning(
+                f"""You have MedCAT version '{__version__}' installed while the CDB was exported by MedCAT version '{cdb_medcat_version}',
+which would still be fine. If you experience any compatibility issues, please reinstall MedCAT
+or download the compatible model."""
+            )
```

Comment:
can we have another log.warning if we try and load different minor and major versions? As this only warns if patch versions are different.
For different minor / major versions maybe use log.warning / log.error respectively?

Reply:
The current warning is based on both the major and the minor versions, which `[:2]` will return. So not sure what that means and are you suggesting warning on the patch version difference as well?

Reply:
oops sorry misread this as `[2:]` not `[:2]` - ha.

That warn message isn't quite right then, MedCAT generally isn't going to work between major versions, and even between minor versions we should say something may not work, not that "would still be fine"

Reply:
Cool. Got it reworded. Reluctant to use log.error here due to the weeny possibility that an older model may work with a newer release.